test/backend_gather.in.cpp

```diff
@@ -36,6 +36,78 @@ using namespace ngraph;
 
 static string s_manifest = "${MANIFEST}";
 
+NGRAPH_TEST(${BACKEND_NAME}, gather_4d_indices_no_axis)
```

Comment:
does 4d here refer to rank of indices or the number 4 in indices?

Reply:
My mistake, should be 3d. It's the rank of indices.